src/renderer/store/exportIo/actions.ts

```diff
@@ -1,19 +1,13 @@
-import { RuntimeProject } from 'sharedTypes';
+import { RuntimeProject } from '../../../sharedTypes';
 import { Action } from '../action';
 
 export const START_EXPORT = 'START_EXPORT';
 export const EXPORT_PROGRESS_UPDATE = 'EXPORT_PROGRESS_UPDATE';
 export const FINISH_EXPORT = 'FINISH_UPDATE';
 
-export const startExport: (
-  projectId: string,
-  exportFilePath: string
-) => Action<{
-  projectId: string;
-  exportFilePath: string;
-}> = (projectId, exportFilePath) => ({
+export const startExport: () => Action<any> = () => ({
```

Comment:
don't use `any` here - if the payload is empty then it's `Action<null>`

Reply:
For some reason, if I change it to `null` the `startExport()` will not be executed, so I added `projectId` and `filePath` back